sdk/storage/Cargo.toml

```diff
@@ -48,6 +48,7 @@ url = "2.1"
 default = ["account", "blob", "queue", "table"]
 test_e2e = ["adls_gen2", "account", "blob", "queue", "table"]
 account = []
+azurite_workaround = []
```

Comment:
Question: what's this feature for? I don't see it being used anywhere.

Reply:
It's used basically here: https://github.com/Azure/azure-sdk-for-rust/blob/85e37c2a3765af88873ed9cbd7ac072e920a6127/sdk/storage/src/blob/blob/mod.rs#L110-L113

Turns out azurite is bugged and does not conform to the Azure specifications (it does not return a valid creation time). That caused the SDK to return errors rendering it unusable with Azurite.

We added a feature flag to allow the exception for azurite (without impacting the Azure cloud users).

Now that I think about it's probably better to have the `azurite_workaround` feature to include `blob`... what do you think?

Reply:
Yea, if this feature is only exposed in blob, then that makes sense.